jared-alrightjones/Homework-4/madlib-with-loops/js/app.js

```diff
@@ -0,0 +1,24 @@
+// provided code
+
+var startupX = ['Uber', 'Google', 'Amazon', 'Apple', 'Facebook', 'Twitter'];
+var startupY = ['Slack', 'GitHub', 'Tesla', 'Hyperloop', 'Harvest'];
+
+var random1 = Math.floor((Math.random() * startupX.length));
+var random2 = Math.floor((Math.random() * startupY.length));
+
+// create something
+
+var greatIdea;
+const createBtn = document.querySelector('#create'); 
+const outputField = document.querySelector('#xForY');
+createBtn.addEventListener('click' outputText)
```

Comment:
Notice your `addEventListener` method has two arguments here, but they're separated only by a space, which won't work. What character do we use to separate arguments when calling a method?